MetroscopeModelingLibrary/Tests/Multifluid/HeatExchangers/CoolingTower_reverse.mo

```diff
@@ -32,7 +32,7 @@ model CoolingTower_reverse
   output Real Q_cold(start=52552.133) "m3/s";
 
   output Real airOutletPress(start=1) "bar";
-  output Real AirOutletTemp(start=35) "deg_C";
+  input Real AirOutletTemp(start=35) "deg_C";                            //output
```

Comment:
With the correction on the component (isoPH instead of isoP), you won't have anymore to impose this temperature